packages/terra-search-field/src/SearchField.jsx

```diff
@@ -63,11 +74,17 @@ class SearchField extends React.Component {
   handleTextChange(event) {
     this.setState({ searchText: event.target.value });
 
-    if (!this.searchTimeout) {
+    if (!this.searchTimeout && !this.props.disableAutoSearch) {
       this.searchTimeout = setTimeout(this.handleSearch, this.props.searchDelay);
     }
   }
 
+  handleKeyDown(event) {
+    if (event.nativeEvent.keyCode === KEYCODES.ENTER) {
```

Comment:
Is there any difference between event.nativeEvent.keyCode and event.keyCode? I see that some components are using event.nativeEvent.keyCode while others are using event.keyCode

Reply:
`event.nativeEvent` references the browser's native event, while `event` is a React cross browser synthetic event. Seems that React prefers synthetic events to standardize handling events across browsers.